px-data-grid.html

```diff
@@ -417,8 +423,16 @@
           );
         }
 
+        _readValue(column, item) {
```

Comment:
@alump, We already have _getValue method, can it be used for your purposes?

Reply:
_readValue has extra functionality required for now (as far as items are used). But _getItems usage can be replaced with _readValue usage. And if this is done, it's ok to rename _readValue to _getValue.